pystats_docs.md

```diff
@@ -0,0 +1,81 @@
+# pystats documentation
+
+This describes the meaning of the various sections and fields in the pystats output.
+
+## Execution counts
+
+This is the counts of how many times each Tier 1 instruction is executed.
+
+The "miss ratio" column shows the percentage of times the instruction executed that it deoptimized.
+
+## Pair counts
+
+Shows the Top 100 most frequently occurring pairs of instructions as executed.
+
+## Predecessor / Successor pairs
+
+Shows the top 5 instructions that are executed before and after each instruction.
+
+## Specialization stats
+
+TBD
+
+## Specialization effectiveness
+
+TBD
+
+## Call stats
+
+TBD
+
+## Object stats
+
+TBD
+
+## GC stats
+
+TBD
+
+## Optimization (Tier 2) stats
+
+- Optimization attempts: The number of times a trace is found, and attempt is
```

Comment:
Traces aren’t really “found”. I assume this just counts either calls to the optimizer or just how many times we hit JUMP_BACKWARD with the counter high (or low?) enough?

Reply:
It's the latter (how many times we hit JUMP_BACKWARD with the counter high (or low?) enough).  I'll update this to reflect that.